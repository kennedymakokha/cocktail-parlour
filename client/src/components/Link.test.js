import React from 'react';
import Link from './Index.react'
import renderer from 'react-test-renderer';
 
it('renders correctly', () => {
 const tree = renderer
   .create(<Link page="kennesy" />)
   .toJSON();
   console.log(tree)
   expect(tree).toMatchSnapshot();
});