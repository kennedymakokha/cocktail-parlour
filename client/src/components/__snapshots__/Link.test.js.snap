// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="normal"
  href="kennesy"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
/>
`;
